provider/aws_sd_test.go

```diff
@@ -722,23 +722,31 @@ func TestAWSSDProvider_RegisterInstance(t *testing.T) {
 		},
 	}
 
-	// ALIAS instance
+	// AWS ELB instance (ALIAS)
 	provider.RegisterInstance(services["private"]["alias-srv"], &endpoint.Endpoint{
 		RecordType: endpoint.RecordTypeCNAME,
 		DNSName:    "service1.private.com.",
 		RecordTTL:  300,
-		Targets:    endpoint.Targets{"load-balancer.us-east-1.elb.amazonaws.com", "load-balancer.us-west-2.elb.amazonaws.com"},
```

Comment:
Why are you modifying this test case, instead of creating a separate one?

Reply:
I wasn't sure if it is necessary to test two load-balancers of the same type. But agree. Reverted that change and there is a new test case for NLB.